UserManagement/src/main/java/com/flmfoods/model/User.java

```diff
@@ -0,0 +1,36 @@
+package com.flmfoods.model;
+
+import java.util.List;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name="users")
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class User {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private long userId;
+	private String userName;
+	private String phoneNum;
+	private String email;
+	private String password;
+	@OneToMany
+	@JoinColumn(name="address_id")
+	private List<Address> address;
+	@OneToMany(cascade=CascadeType.ALL,mappedBy="user")
+	private List<Order> orders;
```

Comment:
```suggestion
	private List<Order> orders;
```
I feel we can remove `Orders` from here, we will handle them in Order class by having `userId` there